Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1824,6 +1850,17 @@ extension BrowserViewController: ReaderModeDelegate, UIPopoverPresentationContro
     }
 }
 
+// MARK: - UIPopoverPresentationControllerDelegate
+
+extension BrowserViewController: UIPopoverPresentationControllerDelegate {
+    func popoverPresentationControllerDidDismissPopover(popoverPresentationController: UIPopoverPresentationController) {
+        displayedPopoverController = nil
```

Comment:
I just thought that I still need to keep a reference to `displayedPopoverController`as I dismiss it then need redisplay it after the rotation/size changes.
So I'll keep this as it is.